argusclient/client.py

```diff
@@ -416,7 +416,18 @@ def add(self, alert):
             alertobj.trigger = alertobj.triggers.add(alert.trigger)
         if alert.notification:
             alertobj.notification = alertobj.notifications.add(alert.notification)
-        if alert.trigger and alert.notification:
+        elif alert.notifications and isinstance(alert.notifications, list):
+            notifications = []
+            for notification in alert.notifications:
+                notifications.append(alertobj.notifications.add(notification))
+            alertobj.notifications = notifications
```

Comment:
`alertobj.notifications` is expected to be a service object of type `AlertNotificationsServiceClient`, so this will break the contract. You can however use the notifications list to initialize the collection that is behind the service object so that it will avoid one API call.